### ISSUE TITLE
[opentelemetry-kube-stack] `scrape_configs_file` is basically useless

### ISSUE THREAD

**Issue by kristeey** [open-telemetry/opentelemetry-helm-charts#1331]
Hi,
Trying out different features in the `opentelemetry-kube-stack` helm chart, and came over the `scrape_configs_file` attribute in the `collectors.collectorName` section. This is basically useless unless the user wants to use the default scrape config that comes with the helm chart `deamon_scrape_configs.yaml` or the user clones the chart and installes the helm chart locally. When installing the helm chart using the remote repository it is not possible to include "external" scrape config files into the helm structure (because of helm...). This is also the case when the chart is used as a subchart, and the scrape config files exists in the parent chart. Quoting the [helm docs](https://helm.sh/docs/chart_template_guide/accessing_files/):

> "Files outside of a Helm application [subchart](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/), including those of the parent, cannot be accessed"

this is a limitation with the `.Files` object in Helm.

**Comment by jaronoff97** [comment]
Yes, this is a very annoying limitation of helm, this is useful if and only if you are forking the chart with a local copy. I'm not sure there's a way around this unfortunately. FWIW, I run the chart as a fork and periodically update my local copy for this reason.
